lib/puma/log_writer.rb

```diff
@@ -59,6 +59,8 @@ def log(str)
       @stdout.puts(format(str)) if @stdout.respond_to? :puts
 
       @stdout.flush unless @stdout.sync
+
+      @custom_logger.write(format(str)) if @custom_logger
```

Comment:
I think we would want to write to the custom logger _instead_ of stdout, right?

Reply:
Sorry for the late reply - I think that makes sense, added the change.